spec/controllers/post_spec.rb

```diff
@@ -105,14 +105,7 @@
         id: 1,
         validation: "0f21473d03145662d38ce4ea1ebac790"
       )
-      get :confirm, validation: "0f21473d03145662d38ce4ea1ebac791"
-
-      expect(response).to redirect_to validation_error_url
-    end
-
-    it "redirects to error page without validation" do
```

Comment:
Since validation is a URL param now, and we have the `constraints` that it's 32 alphanumeric characters, it won't even properly route without a validation param that matches the regex.

`No route matches {:action=>"confirm", :controller=>"posts", :id=>"4209"}`